### PR TITLE
Add rust environment, and time_complexity.md rust code

### PR DESCRIPTION
**Pull request by xBLACKICEx**
If this PR is related to coding or code translation, please fill out the checklist.

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [x] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

File: docs/chapter_computational_complexity/time_complexity.md [hunk 1]
```diff
@@ -153,6 +153,21 @@ $$
     }
     ```
 
+=== "Rust"
+
+    ```rust title=""
+        // 在某运行平台下
```
Comment (krahets) [inline docs/chapter_computational_complexity/time_complexity.md:159]: In correct indentaiton.

**Review by krahets — COMMENTED** [review]
Thanks! You become the first contributor to coding in Rust.

Could you add codes in `codes/rust/...` at first and not copy them to the docs? Once there are a fair amount of Rust codes, we show them in the docs.

By the way, please check the code carefully and test it. Paste the output in the console to the PR. Thanks~

**Comment by xBLACKICEx** [comment]
@krahets

I think it should be okay this time

**Comment by krahets** [comment]
Thanks! Could you please remove the code in the markdown file? Once there are a fair amount of Rust codes, we show them in the docs.

**Comment by krahets** [comment]
Thanks! Could you please remove the code in the markdown file? Once there are a fair amount of Rust codes, we show them in the docs.

**Review by krahets — APPROVED** [review]
Thank you! Now merge it.

**Pull request merged**